includes/wp-cli/class-events.php

```diff
@@ -434,14 +432,9 @@ private function delete_event_by_id( $args, $assoc_args ) {
 	}
 
 	/**
-	 * Delete all events of the same action
-	 *
-	 * @param array $args Array of positional arguments.
-	 * @param array $assoc_args Array of flags.
+	 * Delete all events of the same action.
 	 */
-	private function delete_event_by_action( $args, $assoc_args ) {
-		$action = $assoc_args['action'];
-
+	private function delete_event_by_action( $action ) {
```

Comment:
Rather greatly just simplified this CLI method. Functionally it works the same, just less verbose. I tested w/ a 25k event deletion, and all went well. There is room to optimize if needed, but this is pretty rarely-used functionality, so I'm not really worried.